chcemvediet/apps/anonymization/cron.py

```diff
@@ -146,3 +142,92 @@ def attachment_normalization():
         normalize_using_imagemagic(attachment)
     else:
         skip_normalization(attachment)
+
+def recognize_using_mock(attachment_normalization):
+    normalized = os.path.join(settings.PROJECT_PATH,
+                              u'chcemvediet/apps/anonymization/mocks/recognized.odt')
+    with open(normalized, u'rb') as file:
+        AttachmentRecognition.objects.create(
+            attachment=attachment_normalization.attachment,
+            successful=True,
+            file=ContentFile(file.read()),
+            content_type=content_types.ODT_CONTENT_TYPE,
+            debug=u'Created using mocked abbyyocr11.'
+        )
+    cron_logger.info(u'Recogized attachment_normalizon using mocked abbyyocr11: {}'.format(
+        attachment_normalization))
+
+def recognize_using_ocr(attachment_normalization):
+    if settings.MOCK_OCR:
+        recognize_using_mock(attachment_normalization)
+        return
+
+    try:
+        p = None
+        with temporary_directory() as directory:
+            filename = os.path.join(directory, u'file.pdf')
+            shutil.copy2(attachment_normalization.file.path, filename)
+            p = subprocess32.run(
+                [u'abbyyocr11', u'--recognitionLanguage', u'Slovak', u'--splitDualPages', u'-if',
```

Comment:
Este som neotestoval `abbyyocr11`, nepodarilo sa mi ho totiz aktivovat:
```
Result:                                                                      
  Your serial number cannot be activated automatically.                        
  You may be trying to activate more copies of the product than is provided for
   by the License Agreement.   
```

Reply:
Riesime to...

Reply:
Pretestoval som to aj s produkcnym `abbyyocr11`. A vyzera ze funguje. Ked budes mat spraveny pristup k produkcnemu `abbyyocr11`, tak si to vyskusas tiez. Aby sme PR nezdrzovali, tak mozeme mergnut a ist dalej.

Reply:
:ok: